editor/editor_node.cpp

```diff
@@ -426,6 +426,12 @@ void EditorNode::_update_from_settings() {
 	}
 	_update_title();
 
+	RD *device = RD::get_singleton();
+	if (device) {
+		const int latency_mode = int(GLOBAL_GET("rendering/rendering_device/vsync/latency_mode")) + 1;
+		device->set_latency_mode((RD::LatencyMode)latency_mode);
+	}
```

Comment:
This makes me wonder if we should have an editor setting for the latency mode, like we do for V-Sync mode already. Most of the time, you'll want the editor to be using a low-latency setting to ensure good UI responsiveness.

In particular, I foresee some users willing to go for the `low_extreme` latency mode, which isn't available in the project settings but could be made available in the editor settings. Of course, if we decide to make it possible, we'll have to add a warning label that's displayed below the FPS counter in the 3D editor's View Frame Time panel.